web/packages/extension/webpack.config.js

```diff
@@ -41,11 +42,7 @@ function transformManifest(content, env) {
     // when it gets generated in web/packages/core/tools/set_version.js and then
     // load it in the code above.
 
-    // The extension marketplaces require the version to monotonically increase,
-    // so append the build number onto the end of the manifest version.
-    manifest.version = buildId
-        ? `${packageVersion}.${buildId}`
-        : packageVersion;
+    manifest.version = packageVersion;
```

Comment:
Is this handled somewhere else now?

Reply:
This will have to be rewritten, because I assumed I can use semver in the extension (these versions are unique and ordered). The problem is similar to wix's, we have to somehow translate nightly `a.b.c-nightly.d.e.f` versions to `a.b.c.d` here.

I can use `github.run_number` for that (like before) or something like number of days since a point in time. But I think it's wise to wait with that for @Korne127's PR and improved Windows metadata to make a decision.

Reply:
We'll switch the extension to stable releases, so we don't need to handle nightly here

Reply:
But we'll publish extensions anyway in nightly releases on GH, wouldn't that be a problem? (Probably not if the version requirement is only for extension stores and not extensions installed locally)

Reply:
Yeah I don't think it'll be an issue, it's a store requirement. Will need to test it of course though.

And we *may* eventually want to make a separate nightly extension submission, to match what we have today - not sure. It's unlikely though.

Reply:
Hmmm... will we need to provide `a.b.c.d` versions for wix though? You know, to preserve version ordering when users install nightly releases. If yes, then I guess it costs nothing to do the same for extensions.

Reply:
Yeah for windows builds we need to stick to `a.b.c.d` number format (for wix/msi and also the exe resource data), and MSI specifically will only test the first three numbers to see "is the version different"

Reply:
I added a version in format `a.b.c.d`, and called it `version4`. It's generated as follows:
1. for stable minor: `1.2.3` (current version) -> `1.3.0` (version) -> `1.3.0` (version4)
2. for nightly: `1.2.3` (current version) -> `1.3.0-nightly.1234.5.6` (version) -> `1.2.3.D` (version4)

Where `D` is a number related to the current day (`year - 2020` concatenated with zero-padded day of the year, so for today the number is 4221).